nipoppy_cli/nipoppy/config/pipeline_step.py

```diff
@@ -0,0 +1,38 @@
+"""Pipeline step configuration."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Optional
+
+from pydantic import ConfigDict, Field
+
+from nipoppy.config.container import SchemaWithContainerConfig
+
+
+class PipelineStepConfig(SchemaWithContainerConfig):
+    """Schema for processing pipeline step configuration."""
+
+    NAME: Optional[str] = Field(
+        default=None,
+        description="Step name, required if the pipeline has multiple steps",
+    )
+    DESCRIPTOR_FILE: Optional[Path] = Field(
```

Comment:
I thought descriptor files were specific to pipeline (i.e. container) and not steps?

Reply:
We clarified that there should be one descriptor per argparse parser (or equivalent, e.g. usage message). This will also be needed for e.g. Micapipe